src/main/kotlin/org/openrewrite/RecipeMarkdownWriter.kt

```diff
@@ -651,17 +651,13 @@ import TabItem from '@theme/TabItem';
     private fun BufferedWriter.writeContributors(recipeDescriptor: RecipeDescriptor) {
         if (recipeDescriptor.contributors.isNotEmpty()) {
             newLine()
-            writeln("## Contributors")
-            writeln(
-                recipeDescriptor.contributors.stream()
-                    .map { contributor: Contributor ->
-                        if (contributor.email.contains("noreply")) {
-                            contributor.name
-                        } else {
-                            "[" + contributor.name + "](mailto:" + contributor.email + ")"
-                        }
-                    }.collect(joining(", "))
-            )
+            writeln("## Contributors\n")
+            // Extract unique contributor names (removing duplicates)
+            val uniqueNames = recipeDescriptor.contributors
+                .map { contributor -> contributor.name }
+                .distinct()
+                .sorted()
```

Comment:
The order or distinct & sorted matters, I think; distinct only removes duplicates that are next to each other.

Apart from that I would not change the order by sorting, as by default we sort by who has contributed the most lines of code(!), which I'd want to retain.

Reply:
My understanding is that `distinct()` removes all duplicates from the collection - not just ones next to each other. 

Good call out on sorted - removed that.